flake8_quotes/__init__.py

```diff
@@ -158,20 +158,25 @@ def get_quotes_errors(self, file_contents):
             last_quote_char = token.string[-1]
             first_quote_index = token.string.index(last_quote_char)
             unprefixed_string = token.string[first_quote_index:]
+            is_multiline_string = '\n' in unprefixed_string
```

Comment:
That's a nice thought but it won't work for multiquote strings that are only 1 line long (e.g. `"""foo"""`). How about this logic?

```python
# (" * 3 -> """) == (""")foo"""
# (" * 3 -> """) == ("fo)o"
unprefixed_string[0] * 3 == unprefixed_string[0:2]
```